src/pushcollector/_impl/proxy.py

```diff
@@ -50,11 +51,48 @@ class CollectorProxy(object):
     def __init__(self, delegate):
         self._delegate = delegate
 
+    def _translate_pushitem(self, pushitem):
+        def _get_checksum():
+            if pushitem.md5sum and pushitem.sha256sum:
```

Comment:
Why do we only save checksums if *both* of these are available?

Reply:
I understood the json schema incorrectly where it looked like either the checksums is null or has both the checksums. Also checking on Pub, it looked like that's how it is.